### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by thibault-martinez**
## Change checklist

Add an `x` to the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/keys/age.rs [hunk 1]
```diff
@@ -31,7 +31,7 @@ const MAC_BASE64_LEN: usize = 43;
 fn derive_wrap_key(password: &[u8], salt: &[u8; 16], work_factor: u8, wrap_key: &mut [u8; 32]) {
     // wrap key = scrypt(N = work factor, r = 8, p = 1, dkLen = 32,
     //     S = "age-encryption.org/v1/scrypt" || salt, P = passphrase)
-    let params = ScryptParams::new(work_factor, 8, 1).unwrap();
+    let params = ScryptParams::new(work_factor, 8, 1, 32).unwrap();
```
Comment (thibault-martinez) [inline src/keys/age.rs:34]: @semenov-vladyslav Just confirming, 32 is the right value?

Comment (semenov-vladyslav) [reply src/keys/age.rs:34]: Right, the derived from password via scrypt wrap key is 32-byte key used to protect file key via ChaCha20-Poly1305.

File: src/macs/hmac.rs [hunk 2]
```diff
@@ -6,26 +6,26 @@
 #[cfg(feature = "sha")]
 #[cfg_attr(docsrs, doc(cfg(feature = "sha")))]
 pub fn HMAC_SHA256(data: &[u8], key: &[u8], mac: &mut [u8; 32]) {
-    use hmac_::Mac;
-    let mut m = hmac_::Hmac::<sha2::Sha256>::new_from_slice(key).unwrap();
+    use hmac::Mac;
```
Comment (semenov-vladyslav) [inline src/macs/hmac.rs:9]: I feel like we shouldn't use Mac trait here, it's going to change in the next version. Should probably use KeyInit and Update/Digest instead.

**Review by semenov-vladyslav — DISMISSED** [review]
Last final update in hmac needed.

File: src/macs/hmac.rs (hunk 2 above)
Comment (semenov-vladyslav) [reply src/macs/hmac.rs:9]: `Mac` trait in [digest-0.11](https://docs.rs/digest/0.11.0-pre/digest/trait.Mac.html) will not have `new` method. Right now there's no need to prepare for that.

**Review by semenov-vladyslav — APPROVED** [review]
Looks good.

File: src/ciphers/aes_cbc.rs [hunk 3]
```diff
@@ -1,141 +1,81 @@
...
 };
 use sha2::{Sha256, Sha384, Sha512};
 use subtle::ConstantTimeEq;
+use zeroize::Zeroize;
 
 use crate::ciphers::traits::{Aead, Key, Nonce, Tag};
 
 /// AES-CBC using 128-bit key and HMAC SHA-256.
-pub type Aes128CbcHmac256 = AesCbc<Aes128, Sha256, U16, U16>;
+pub type Aes128CbcHmac256 = CbcHmac<Aes128, Pkcs7, Sha256>;
```
Comment (thibault-martinez) [inline src/ciphers/aes_cbc.rs:33]: Out of curiosity, was `AesCbc` previously automatically using `Pkcs7`? Or why do we need to put it now?

Comment (semenov-vladyslav) [reply src/ciphers/aes_cbc.rs:33]: Yes, for some reason, although there could be other paddings. Also, it was named AesCbc but it was possible to use ciphers other than AES. Having padding as generic argument just feels natural in the current implementation.

Comment (thibault-martinez) [reply src/ciphers/aes_cbc.rs:33]: Thanks

**Pull request merged**